components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/AbstractUserInfoResponseBuilder.java

```diff
@@ -315,4 +324,15 @@ private List<String> getEssentialClaimUris(OAuth2TokenValidationResponseDTO toke
         }
         return new ArrayList<>();
     }
+
+    private String getCallbackUrl(String clientId) throws UserInfoEndpointException {
+        OAuthAppDO oAuthAppDO;
+        try {
+            oAuthAppDO = OAuth2Util.getAppInformationByClientId(clientId);
+        } catch (IdentityOAuth2Exception | InvalidOAuthClientException e) {
+            throw new UserInfoEndpointException("Error while retrieving OAuth app information for clientId: "
```

Comment:
ny reason to skip e at throw?

Reply:
fixed with 14c704f19c94bbde0f9926ce500b4222b9ebfdaa